### PR TITLE
Indicate use of streams for unordered messages as example

### PR DESCRIPTION
**Pull request by mirjak**
Just a proposal...

### CODE REVIEW

File: draft-ietf-taps-interface.md [hunk 1]
```diff
@@ -1576,7 +1576,8 @@ Default:
 If true, it specifies that the receiver-side transport protocol stack may only deliver the Message to the receiving application after the previous ordered Message which was passed to the same Connection via the Send
 Action, when such a Message exists. If false, the Message may be delivered to the receiving application out of order.
 This property is used for protocols that support preservation of data ordering,
-see {{prop-ordering}}, but allow out-of-order delivery for certain messages.
+see {{prop-ordering}}, but allow out-of-order delivery for certain messages, e.g., by multiplex independent messages onto
```
Comment (mwelzl) [inline draft-ietf-taps-interface.md]: s/multiplex/multiplexing

**Comment by abrunstrom** [comment]
Well, you could do it with streams in SCTP as well so as an example it seems approriate. And good with an example I think.

**Comment by mirjak** [comment]
Yes, we do have actually these kind of examples in many cases in the doc and I just thought when reading one could help here as well.

File: draft-ietf-taps-interface.md (hunk 1 above)
Comment (mwelzl) [reply draft-ietf-taps-interface.md]: done

**Comment by mwelzl** [comment]
I don't understand why one would need streams for this in SCTP, but it doesn't matter. As I said I have no strong opinion, I'm fine with adding this example (and I just fixed the nit).

**Comment by abrunstrom** [comment]
@mwelzl you do not need to, as there are multiple ways to do it in SCTP, but you could.

**Pull request merged**